dts/bindings/sensor/lm35.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: Apache-2.0
+
+description: LM35 Digital Temperature Sensor
+
+compatible: "lm35"
+
+include: sensor-device.yaml
+
+properties:
+  io-channels:
+    required: true
+    description: |
+      ADC channel that will perform measurement
+
+  resolution:
+    type: int
+    default: 12
+    description: |
+      ADC resolution to be used for the channel
```

Comment:
Please add a justification for the default

Reply:
Done. I was quite unsure on how to justify this, if you think on a better comment than the one I added, please let me know.

Reply:
Move the justification into the description so it will be rendered in the documentation